packages/core/utils/src/common/__tests__/define-config.spec.ts

```diff
@@ -256,4 +257,93 @@ describe("defineConfig", function () {
       }
     `)
   })
+
+  it("should not include disabled modules", function () {
```

Comment:
suggestion: should we add a test where we disable a module and therefore it should not be resolvable from the container. wdyt? it could go there integration-tests/modules/__tests__/modules/load-standalone.ts